From c7ca56347e3a4297fefee96792bdec574d967191 Mon Sep 17 00:00:00 2001
From: Yongjun Zhang <yzhang@cloudera.com>
Date: Thu, 19 Nov 2015 14:20:33 -0800
Subject: [PATCH 0984/2748] HDFS-9439. Include status of closeAck into
 exception message in DataNode#run. (Xiao Chen via
 Yongjun Zhang)

(cherry picked from commit 31dc5c650e3fcfd5b581cbc1132b895d3eadaf49)

Change-Id: I267a3e682e6fde404cd72f76925eec0e5aabcd71
---
 .../hadoop/hdfs/server/datanode/DataNode.java      |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/DataNode.java b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/DataNode.java
index 07cebe7..2a09416 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/DataNode.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/DataNode.java
@@ -2176,7 +2176,7 @@ public void run() {
                    + Arrays.asList(targets));
             } else {
               throw new IOException("Bad connect ack, targets="
-                  + Arrays.asList(targets));
+                  + Arrays.asList(targets) + " status=" + closeAck.getStatus());
             }
           }
         }
-- 
1.7.9.5

